nextflow.config

```diff
@@ -60,40 +89,53 @@ params {
     maps_cutoff_fdr            = 2
     maps_filter                = 'None'
     maps_model                 = 'pospoisson'
+    maps_3d_ext                = 'sig3Dinteractions.pe.txt'
     merge_map_py_source        = 'https://raw.githubusercontent.com/ijuric/MAPS/91c9c360092b25a217d91b9ea07eba5dd2ac72f4/bin/utils/genomic_features_generator/scripts/merge_map.py'
     feature_frag2bin_source    = 'https://raw.githubusercontent.com/ijuric/MAPS/91c9c360092b25a217d91b9ea07eba5dd2ac72f4/bin/utils/genomic_features_generator/scripts/feature_frag2bin.py'
     make_maps_runfile_source   = 'https://raw.githubusercontent.com/ijuric/MAPS/18c1a337f222130d7c5735d051614e2a253d5319/bin/MAPS/make_maps_runfile.py'
```

Comment:
why not having the python files in `bin/`?

Reply:
This is an old story. At beginning, I put those files in bin folder. However, during the release v1 review process, they were moved out as a link. By doing this, user can upgrade the scripts by change the url for a quick patch.

Reply:
I see...
I'll ping the @nf-core/core team to have some feedback

Reply:
I will try to copy the scripts soon. The jucier is java bin. Will it be OK to keep as a URL?